projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -329,9 +330,9 @@ function get_subscriber_count( $include_social_followers ) {
 	$counts = fetch_subscriber_counts();
 
 	if ( $include_social_followers ) {
-		$subscriber_count = $counts['value']['email_subscribers'] + $counts['value']['social_followers'];
+		$subscriber_count = $counts['value']['total_subscribers'] + $counts['value']['social_followers'];
 	} else {
-		$subscriber_count = $counts['value']['email_subscribers'];
+		$subscriber_count = $counts['value']['total_subscribers'];
```

Comment:
This is used for Subscribe block but on frontend site (dynamic rendering)

![image](https://github.com/user-attachments/assets/b2c27a86-79cd-45c6-b7bf-3e67a30b4e20)